src/components/Capture/index.ts

```diff
@@ -18,8 +18,7 @@ export const DocumentVideoCapture = appendToTracking(
 )
 
 export const PoACapture = appendToTracking(
-  withCaptureVariant(Document, {
-    isPoA: true,
+  withCaptureVariant(ProofOfAddress, {
     forceCrossDevice: false,
     side: 'front',
```

Comment:
Can we remove `side`? It is not needed for PoA

Reply:
Did it in SA-213.